gateway/src/main/kotlin/org/yapp/gateway/security/SecurityConfig.kt

```
@@ -0,0 +1,58 @@
+        http.csrf { it.disable() }.formLogin { it.disable() }.httpBasic { it.disable() }.logout { it.disable() }
+            .sessionManagement { it.sessionCreationPolicy(SessionCreationPolicy.STATELESS) }.exceptionHandling {
+                it.authenticationEntryPoint(CustomAuthenticationEntryPoint(objectMapper))
+                it.accessDeniedHandler(CustomAccessDeniedHandler(objectMapper))
+            }
+            .oauth2ResourceServer { oauth2 ->
+                oauth2.jwt { jwt ->
+                    jwt.jwtAuthenticationConverter(jwtAuthenticationConverter)
+                }
+            }
+            .authorizeHttpRequests {
+                it.requestMatchers(*WHITELIST_URLS).permitAll()
```

detekt (1.23.8)

[warning] In most cases using a spread operator causes a full copy of the array to be created before calling a method. This may result in a performance penalty.

(detekt.performance.SpreadOperator)